README.md

```diff
@@ -26,11 +28,11 @@ To run this Vagrant VM, you have to install the following tools:
 
 ### Installing the VM
 
-
 1. Make sure you have a reliable internet connection :wink:
-2. On Windows:
+2. Only on Windows (users of a real OS can jump to next step :wink: ):
     * Make sure Hyper-V support is turned off (VirtualBox will not work with HyperV turned on)
-3. Open console in this directory with the downloaded config files
+      * see here for how to do this: https://www.poweronplatforms.com/enable-disable-hyper-v-windows-10-8/
```

Comment:
i find e link wie mä das mache cha wichtig, weil es isch nid eifach zum finde wo mä das usschalte cha

Reply:
i understand the need for a link in your context but i still think it is not necessary here in the base repo